sdk/communication/pom.xml

```diff
@@ -24,27 +24,27 @@
         <dependency>
           <groupId>com.azure</groupId>
           <artifactId>azure-communication-chat</artifactId>
-          <version>1.0.1</version> <!-- {x-version-update;com.azure:azure-communication-chat;current} -->
+          <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-chat;current} -->
```

Comment:
are these changes right? not our components

Reply:
This came from the generation of our library, but you are right that this seems off. I'll revert it for now.

Reply:
Seems safest. Or you can check with an architect to see if they should be these values.

Reply:
You can revert changes to files on other service. This is cause by communication service team not updating their pom correctly to the value of version_client.